src/libraries/Common/src/Interop/Browser/Interop.HttpHandlerService.cs

```diff
@@ -0,0 +1,535 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
Nit: blank line

Reply:
We should have a blank line between the License and first code line?

Reply:
Looked at other code and there should be a blank line.  Thanks

Reply:
addressed in recent commits